plugins/dialog/plugin.js

```diff
@@ -1213,13 +1213,13 @@ CKEDITOR.DIALOG_STATE_BUSY = 2;
 			}
 
 			// Create the HTML for the tab and the content block.
-			var page = CKEDITOR.dom.element.createFromHtml( pageHtml.join( '' ) );
+			var page = CKEDITOR.dom.element.createFromHtml(CKEDITOR.tools.htmlSafeByReview(pageHtml.join( '' ), 'Page is created using vbox.build and inplace internal values') );
 			page.setAttribute( 'role', 'tabpanel' );
 			page.setStyle( 'min-height', '100%' );
 
 			var env = CKEDITOR.env,
 				tabId = 'cke_' + contents.id + '_' + CKEDITOR.tools.getNextNumber(),
-				tab = CKEDITOR.dom.element.createFromHtml( [
+				tab = CKEDITOR.dom.element.createFromHtml( CKEDITOR.tools.htmlSafeByReview( [
 					'<a class="cke_dialog_tab"',
 					( this._.pageCount > 0 ? ' cke_last' : 'cke_first' ),
 					titleHtml,
```

Comment:
titleHTML is htmlEncoded, let's add that to the justification.

Reply:
Added that `titleHtml` is encoded to the justification